Automattic/wp-calypso
# A4A: Display Pressable plan usage information

Resolves: https://github.com/Automattic/automattic-for-agencies-dev/issues/1124

## Proposed Changes

This PR adds the current Pressable plan usage overview to the Marketplace > hosting section:

**Desktop**
![image](https://github.com/user-attachments/assets/f42bf800-4792-451f-8c68-2ac047b2ed29)

**Mobile**
![image](https://github.com/user-attachments/assets/30812d75-f742-4d8e-a616-6a4d45bb93f4)

**Purchases > Licenses section**

![image](https://github.com/user-attachments/assets/81559e8f-e755-4ef8-bad2-19658a0c7584)


**Agency without Pressable usage data**

If an A4A Pressable agency doesn't have usage data, for instance, the first day because the usage information is not available until the next day, we have to display this case with `?` as value

![image](https://github.com/user-attachments/assets/8decee57-c1b0-4bfc-85b2-1eaa4be67af1)


## Testing Instructions

- Check the code
- Launch a horizon live site and check the design
- I hardcoded the usage data for testing purposes, but once it's reviewed and before merge this, I'll get the data from the agency object.

## Pre-merge Checklist
